geode-for-redis/src/main/java/org/apache/geode/redis/internal/commands/executor/server/CommandCommandExecutor.java

```diff
@@ -39,7 +39,7 @@
  * COMMAND refers to the redis command being implemented.
```

Comment:
this javadoc needs updating since "COMMAND" is no longer part of the class name

Reply:
Updated.